frontend/src/lib/query/createTableColumnsFromQuery.ts

```diff
@@ -570,6 +572,29 @@ export const createTableColumnsFromQuery: CreateTableDataFromQuery = ({
 		a.queryName < b.queryName ? -1 : 1,
 	);
 
+	// the reason we need this is because the filling of values in rows doesn't account for mismatch enteries
+	// in the response. Example : Series A -> [label1, label2] and Series B -> [label2,label1] this isn't accounted for
+	sortedQueryTableData.forEach((q) => {
```

Comment:
Isn't the data returned in response already sorted ?

Reply:
no @YounixM it has a custom sorting based on user entry which won't work for us